packages/app/src/public-types.ts

```diff
@@ -139,6 +182,66 @@ export interface FirebaseAppSettings {
   automaticDataCollectionEnabled?: boolean;
 }
 
+/**
+ * @public
+ *
+ * Configuration options given to {@link (initializeServerApp:1) | initializeServerApp()}
+ */
+export interface FirebaseServerAppSettings extends FirebaseAppSettings {
+  /**
+   * An optional Auth ID token used to resume a signed in user session from a client
+   * runtime environment.
+   *
+   * If provided, the FirebaseServerApp instance will work to validate the token even before Auth
+   * is initialized. The result of the validation can be queried via by the application by invoking
+   * {@link (FirebaseServerApp.authIdTokenVerified()}. Awaiting the Promise returned by
+   * {@link (FirebaseServerApp.authIdTokenVerified()} is highly recommended if an authIdToken token
+   * is provided.
+   *
+   * Invoking getAuth() with a FirebaseServerApp configured with a validated authIdToken will cause
+   * an automatic attempt to sign in the user that the authIdToken represents. The token
+   * needs to have been recently minted for this operation to succeed, otherwise it will fail
+   * validation.
+   *
+   * If the token fails local verification, or if the Auth service has deemed it invalid when
```

Comment:
Is there a less anthropomorphic term we could use?  Is "deemed invalid" basically "failed to validate"?

Reply:
Done.